xbstrap/base.py

```diff
@@ -1654,6 +1654,7 @@ def substitute(varname):
     for yml in manifest["tools"]:
         prefix_dir = os.path.join(build_root, yml["prefix_subdir"])
         path_dirs.append(os.path.join(prefix_dir, "bin"))
+        path_dirs.append(os.path.join(prefix_dir, "usr/bin"))
```

Comment:
this is wrong, though, as /usr is not any prefix-derived directory.  if a tool is installing to $PREFIX/usr/bin, it's broken.  see also [`(standards)Directory Variables`](https://www.gnu.org/prep/standards/html_node/Directory-Variables.html).